src/Pos/Wallet/Web/Api.hs

```diff
@@ -257,20 +256,13 @@ type UpdateTx =
     :> ReqBody '[JSON] CTxMeta
     :> WRes Post ()
 
-type GetHistory =
```

Comment:
@flyingleafe no. However, the [search](https://github.com/input-output-hk/daedalus/search?utf8=%E2%9C%93&q=getHistory&type=) in the repository yields no results, and `daedalus-bridge` can be built successfully. Is there anything I'm missing here?